### PR TITLE
8340336: Open some checkbox awt tests

### PR DESCRIPTION
**Pull request by alisenchung**
Fourth set of dialog swing tests to open during fall 2024 test sprint

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340336](https://bugs.openjdk.org/browse/JDK-8340336): Open some checkbox awt tests (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21433/head:pull/21433` \
`$ git checkout pull/21433`

Update a local copy of the PR: \
`$ git checkout pull/21433` \
`$ git pull https://git.openjdk.org/jdk.git pull/21433/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21433`

View PR using the GUI difftool: \
`$ git pr show -t 21433`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21433.diff">https://git.openjdk.org/jdk/pull/21433.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21433#issuecomment-2403668678)

### CODE REVIEW

**Comment by honkar-jdk** [comment]
@alisenchung JBS associated with this PR is- "8340336: Open some checkbox awt tests", but these aren't checkbox tests. Please check there is a mismatch with the tests and the associated JBS.

File: test/jdk/java/awt/Dialog/ClosingParentTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowAdapter;
+
+/*
+ * @test
+ * @bug 4336913
+ * @summary On Win32, disable parent window controls while modal dialog is being created.
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
[the remark this answers is not in the document]
Comment (alisenchung) [reply test/jdk/java/awt/Dialog/ClosingParentTest.java]: wrong set of tests, oops!

**Review by prrace — APPROVED** [review]

**Review by honkar-jdk — APPROVED** [review]
LGTM

**Pull request closed without merging**